server/sonar-qa-util/src/main/java/org/sonarqube/qa/util/pageobjects/SystemInfoPageItem.java

```diff
@@ -33,51 +30,51 @@ public SystemInfoPageItem(SelenideElement elt) {
     }
 
   public SystemInfoPageItem shouldHaveHealth() {
-    elt.$(".system-info-health-info .status-indicator").should(exist);
+    elt.$(".system-info-health-info .status-indicator").should(Condition.exist);
     return this;
   }
 
   public SystemInfoPageItem shouldHaveSection(String section) {
     ensureOpen();
-    elt.$$("h4").findBy(text(section)).should(exist);
+    elt.$$("h4").findBy(Condition.text(section)).should(Condition.exist);
     return this;
   }
 
   public SystemInfoPageItem shouldNotHaveSection(String section) {
     ensureOpen();
-    elt.$$("h4").findBy(text(section)).shouldNot(exist);
+    elt.$$("h4").findBy(Condition.text(section)).shouldNot(Condition.exist);
     return this;
   }
 
   public SystemInfoPageItem shouldHaveMainSection() {
     ensureOpen();
-    elt.$$(".system-info-section").get(0).find("h4").shouldNot(exist);
+    elt.$$(".system-info-section").get(0).find("h4").shouldNot(Condition.exist);
     return this;
   }
 
   public SystemInfoPageItem shouldHaveField(String field) {
     ensureOpen();
-    elt.$$(".system-info-section-item-name").findBy(text(field)).should(exist);
+    elt.$$(".system-info-section-item-name").findBy(Condition.text(field)).should(Condition.exist);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal ".system-info-section-item-name" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)